spyder/plugins/pylint/widgets/pylintgui.py

```diff
@@ -310,8 +312,16 @@ def show_log(self):
             TextEditor(self.output, title=_("Pylint output"),
                        readonly=True, size=(700, 500)).exec_()
 
+    @Slot()
+    def analyze_button_handler(self):
+        """ Try to start code analysis when Analyze button pressed."""
+

```

Comment:
Please remove this blank line.

Reply:
Not as embarrassing, but will try to remember it. (But blank after multi-line doc-strings, right?)

Reply:
I realize I interpreted the blank line before method in a class as blank line after class doc-string. So no.